cellprofiler/modules/makeprojection.py

```diff
@@ -1,11 +1,13 @@
 # coding=utf-8
 
 """
-**Make Projection** combines several two-dimensional images of the same
-field of view together, either by performing a mathematical operation
+MakeProjection
+==============
+**MakeProjection** combines several two-dimensional images of the same
+field of view by performing a mathematical operation
 upon the pixel values at each pixel position.
 
-This module combines a set of images by performing a mathematic
+This module combines a set of images by performing a mathematical
 operation of your choice at each pixel position; please refer to the
 settings help for more information on the available operations. The
 process of averaging or summing a Z-stack (3D image stack) is known as
```

Comment:
At the end of this paragraph, for sake of clarification please add:

**MakeProjection** is meant to be used only on single image Z-slices across image cycles; it will not work on images that have been loaded as 3D volumes in **NamesAndTypes**.